openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```
@@ -116,3 +116,15 @@ instances:
     #
     # extra_headers:
     #   <HEADER_NAME>: <HEADER_VALUE>
+
+    ## @param bearer_token_auth - boolean - optional - default: false
+    ## Set bearer_token_auth to true to add kubernetes service account bearer token in HTTP headers for authentication.
```

I think no need to do a reference to kubernetes here. Bearer token authentication is not a Kubernetes concept

```suggestion
    ## Set bearer_token_auth to true to add bearer token authentication header.
```